pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -372,3 +380,37 @@ func (o *LoginOptions) serverProvided() bool {
 	_, err := o.ClientConfig.ClientConfig()
 	return err == nil || !clientcmd.IsNoServerFound(err)
 }
+
+func relativizeClientConfigPaths(cfg *clientcmdapi.Config, base string) (err error) {
```

Comment:
I sort of like the pattern we came up with for master/node config... gather the vars that are file references in a single function, then you can pass those to a generic `resolve` or `relativize` helper... cuts out a lot of the boilerplate